server/db/index.js

```diff
@@ -1,58 +1,71 @@
 const db = require('./db');
 
 //import all models
-const Order = require('./models/Order');
-const OrderDrink = require('./models/Order_Drink');
-const Menu = require('./models/Menu');
-const Employee = require('./models/Employee');
 const Drink = require('./models/Drink');
-const Tab = require('./models/Tab');
+const Employee = require('./models/Employee');
+const Menu = require('./models/Menu');
+const Order = require('./models/Order');
+const OrderDrink = require('./models/OrderDrink');
 const PickUpLocation = require('./models/PickUpLocation');
-const Venue = require('./models/Venue');
-const User = require('./models/User');
+const Tab = require('./models/Tab');
 const TabDrink = require('./models/TabDrink');
-
-//model associations
+const User = require('./models/User');
+const Venue = require('./models/Venue');
 
 //model associations
 
 // OrderDrink is a connection table that connects a Drink to an Order and keeps the quantity and price at the time of order
-// Order.belongsToMany(Drink, { through: OrderDrink });
-// Drink.belongsToMany(Order, { through: OrderDrink });
+Order.belongsToMany(Drink, { through: OrderDrink });
+Drink.belongsToMany(Order, { through: OrderDrink });
 Order.hasMany(OrderDrink);
 OrderDrink.belongsTo(Order);
-// Drink.hasMany(OrderDrink);
-// OrderDrink.belongsTo(Drink);
-// Employee.belongsTo(Venue)
-// Menu.belongsTo(Venue)
-Drink.belongsTo(Menu);
-Menu.hasMany(Drink);
-Tab.belongsTo(Venue);
-Venue.hasMany(Tab);
-Tab.belongsTo(User);
-User.hasMany(Tab);
-PickUpLocation.belongsTo(Venue);
-Venue.hasMany(PickUpLocation);
-TabDrink.belongsTo(Tab);
-Tab.hasMany(TabDrink);
+Drink.hasMany(OrderDrink);
+OrderDrink.belongsTo(Drink);
 
 // the below 2 through associations are causing an error
-// Tab.belongsToMany(Drink, { through: TabDrink });
-// Drink.belongsToMany(Tab, { through: TabDrink });
+Tab.belongsToMany(Drink, { through: TabDrink });
+Drink.belongsToMany(Tab, { through: TabDrink });
+Tab.hasMany(TabDrink);
+TabDrink.belongsTo(Tab);
 Drink.hasMany(TabDrink);
 TabDrink.belongsTo(Drink);
 
+User.hasMany(Order);
+Order.belongsTo(User);
+
+User.hasMany(Tab);
+Tab.belongsTo(User);
+
+Venue.hasMany(Order);
+Order.belongsTo(Venue);
+
+Venue.hasMany(Tab);
+Tab.belongsTo(Venue);
+
+Venue.hasMany(Employee);
+Employee.belongsTo(Venue);
+
+Venue.hasMany(PickUpLocation);
+PickUpLocation.belongsTo(Venue);
+
+Venue.hasMany(Menu);
+Menu.belongsTo(Venue);
+
+Menu.hasMany(Drink);
+Drink.belongsTo(Menu);
```

Comment:
Added some separation between groups here and some ordering so they should be easier to review